mlrun/feature_store/common.py

```diff
@@ -234,6 +235,7 @@ def __init__(
         :param code:        function source code (as string)
         :param requirements: python requirements file path or list of packages
         :param extra_spec:  additional dict with function spec fields/values to add to the function
+        :param auth_info:   used when running on server side
```

Comment:
```suggestion
        :param auth_info:   authentication info. *For internal use* when running on server side
```

That's user-facing doc. As there's no flow that we expect the user to run which actually uses this parameter, let's specify this directly.